docs/how-to/back-up-and-restore.md

```diff
@@ -1,115 +0,0 @@
-(how-to-back-up-and-restore)=
-# Backup and restore
```

Comment:
Question: Are we planning to replace the backup guide with anything? I know it probably doesn't make much sense to back up the data in Kafka, and we no longer have ZooKeeper to store metadata. But not having any How to backup guide feels wrong. To the point, that if we decide to not have a guide on backups, I'd advise adding an explanation page about backups best practice and why we need no guide.

Reply:
Are we able to backlog that item in our docs JIRA? It's a good idea.

Reply:
Yes, both options are viable. Although, I'm not sure whether we want to just call replication guide as a backup guide (backup is usually stored as a file, not as a cluster).
My idea is to create an Explanation page called Backups, where we explain why we usually don't care that much about backing up the data in Kafka and a reference to the cluster migration guide. Such an explanation page shouldn't be long and won't take too long to create. I can do it myself, if you want.

Reply:
> Explanation page called Backups, where we explain why we usually don't care that much about backing up the data in Kafka and a reference to the cluster migration guide

I'm pro that, go for it!

Reply:
Added in https://github.com/canonical/kafka-operator/pull/403/commits/50c05df312f5d583e56bd1f2fe438ac63df88c59, please review.